index.js

```diff
@@ -186,6 +186,19 @@ app.on('ready', () => {
 				},
 			];
 
+			const darkModeToggle = {
+				label: 'Toggle Dark Mode',
+				accelerator: 'CommandorControl+Shift+L',
+				type: 'checkbox',
+				checked: store.get('darkMode', false),
+				click: () => {
+					store.set('darkMode', !store.get('darkMode', false));
+					setTimeout(() => {
+						window.reload();
```

Comment:
@eshack94 Does the toggle only work with a refresh?

When we reload the window we automatically start a new empty chat thread and reset the panes as well, so ideally those remain separate actions if possible.

Reply:
I've only tested with the window reload so far. I can try to figure out another way to handle it, but it might take some trial-and-error and research. Do you have any ideas on alternative ways to handle it?